__tests__/plugin.test.js

```diff
@@ -559,4 +559,59 @@ describe('babel plugin', () => {
     const { code } = runPlugin(input);
     expect(code).toMatchSnapshot();
   });
+
+  it('shows a warning if user uses .value inside inline style, no functions', () => {
+    const input = `
+    function App(){
```

Comment:
```suggestion
    function App (){
```
These code snippets need formatting, please throw them into Prettier.